feat(Pointer): teleport timer

gpvigano opened the pull request:
The current behaviour of the teleport, when the Activate Delay Timer is not 0,
can lead to feel like it is not working (tested with the Pointer Toggle Button
 set to "Touchpad_touch" and  thePointer Set Button set to "Touchpad_Press").

Some users sometimes double-click the teleport button, thus if no timer is set
the teleport is activated twice.

Furthermore when the button is released the beam cannot be activated soon
because of the timer and user feels like that the button is not responding.

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0ccd590afcc3130a94f4e1c7ed8e4d0a5af3711d
  * Your commit message body contains a line that is longer than 72 characters

 * Commit: 0d60a50572c5e2cd245aedf43be4dffd7b8f6124
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/thestonefox/SteamVR_Unity_Toolkit

---
This message was auto-generated by https://gitcop.com

thestonefox commented:
I'd merge these two commits into 1 and i wouldn't consider it a feature, it seems more like a `fix`

gpvigano commented:
Thank you for your reply, I'm sorry for the mistakes in the commit 
message (I am new to GitHub), I hope I'll do better in future.


StoneFox wrote:
> I'd merge these two commits into 1 and i wouldn't consider it a feature,
> it seems more like a |fix|
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/thestonefox/SteamVR_Unity_Toolkit/pull/282#issuecomment-233596130>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATW1qZAq0OFp6SGwRz6C7P27uZumaN5nks5qXKrogaJpZM4JPlat>.
>

thestonefox commented:
@gpvigano no worries, let me know if you need any help with git :)

gpvigano commented:
Well... yes, I need you help right now: if I want to open a new branch, 
but I want to start from my last commits on the feat/teleport_timer 
branch (instead of the master branch), can I branch from 
feat/teleport_timer or it's better to wait the pull on the master and 
then branch from there?
Than you very much for your help.

StoneFox wrote:
> @gpvigano <https://github.com/gpvigano> no worries, let me know if you
> need any help with git :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thestonefox/SteamVR_Unity_Toolkit/pull/282#issuecomment-233622769>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATW1qbcIzcOrf9hCd2sGa57vymrxoEWSks5qXMjQgaJpZM4JPlat>.
>

thestonefox commented:
you don't need to start a new branch if you want to merge your commits.

just do a git rebase and then squash your last 2 commits into one

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0ccd590afcc3130a94f4e1c7ed8e4d0a5af3711d
  * Your commit message body contains a line that is longer than 72 characters

 * Commit: 0d60a50572c5e2cd245aedf43be4dffd7b8f6124
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/thestonefox/SteamVR_Unity_Toolkit

---
This message was auto-generated by https://gitcop.com

gpvigano commented:
Sorry, even after reading some documentation about Git I didn't manage 
to rebase, so I deleted and rebuilt the branch (with a different name).

I am using SourceTree as GUI frontend to Git, I still must undestand how 
the Interactive Rebase feature works...

Anyway now I added a new pull request, later I will start a new branch 
to propose some modifications to the teleport needed for my work, but 
this is another topic.

StoneFox wrote:
> you don't need to start a new branch if you want to merge your commits.
>
> just do a git rebase and then squash your last 2 commits into one
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thestonefox/SteamVR_Unity_Toolkit/pull/282#issuecomment-233625848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATW1qTuE0AQAzA__c-mbuNNjUxd9xP27ks5qXMuGgaJpZM4JPlat>.
>
